src/resources/PowerShellWorkerStrings.resx

```diff
@@ -1,17 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
 <root>
-  <!--
-    Microsoft ResX Schema
-
+  <!-- 
```

Comment:
Please revert all the extra trailing spaces introduced in this file.

Reply:
These spaces are auto formatted